app/views/admin/daily_questions/_form.html.haml

```diff
@@ -0,0 +1,11 @@
+.field
+.field
+  %h4= f.label 'Question'
+  = f.text_field :question_id, class: 'form-control', id: 'Question', required: true
+.field
+  %h4= f.label 'Programmée le'
+  = f.text_field :scheduled_at, class: 'form-control', id: 'Programmée le', required: true
```

Comment:
A date picker, [like this one for example](https://bootstrap-datepicker.readthedocs.io/en/latest/), would be great here!